resources/assets/components/WithReviewing/index.js

```diff
@@ -76,12 +76,22 @@ const reviewComponent = (Component, data) => {
     }
 
     // Open the history modal of the given post
-    showHistory(postId, event) {
-      event.preventDefault()
+    showHistory(postId, event, signupId) {
```

Comment:
Not needed for this pull request, but we should see if we can avoid duplicating this in the `Signup` as well.